pkg/controller/volume/expand/pvc_populator.go

```diff
@@ -80,6 +80,7 @@ func (populator *pvcPopulator) Sync() {
 			glog.V(5).Infof("Error getting persistent volume for pvc %q : %v", pvc.UID, err)
 			continue
 		}
+		// We are only going to add PVCs which are bound and for which pvc.Spec.Size > pvc.Status.Size
```

Comment:
Can we clarify here that the check `pvc.Spec.Size > pvc.Status.Size` is preformed by `AddPVCUpdate`

Reply:
yeah sure.